doc/tutorials/07-electrokinetics/scripts/eof_electrokinetics.py

```diff
@@ -46,7 +46,7 @@
 density_counterions  = -2.0 * sigma / width
 counterions = electrokinetics.Species(density=density_counterions,
                                       D=D, valency=valency,
-                                      ext_force_density=[ext_force_density, 0, 0])
+                                      ext_force_density=[ext_force, 0, 0])
```

Comment:
merge regression?

Reply:
I thought its fine to have the variable just called ext_force. Is an intentional change.

Reply:
But why?

Reply:
Changed it.

Reply:
thanks